### ISSUE TITLE
Recent change adding non-printable characters to jotta-cli status redirected output

### ISSUE THREAD

**Issue by Swallowtail23** [jotta/jotta-cli-issues#103]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**

-------------------------------------------
jottad executable :
jottad appdata    : /var/lib/jottad
jottad version    : 0.6.21799
jotta-cli version : 0.6.21799
-------------------------------------------

**Description of problem:**

Recent changes to jotta-cli status output mean that this line in a bash script:

`jotta-cli status > /root/jotta-update.txt
`
Generates a file with this at the end (when viewed in a text editor):

```
   Status    : Up to date!

------------------------------------------------------------------------------
^[[?25hOK
```

As a result, attaching this as input to an email client (I'm using mailx) sends the text output as an "octet stream" not as text in the email body.

Whilst I can filter it out with tr, it seems more sensible that jotta-status should not be adding the non-printable sequences.

**Expected:**

jotta-cli status when redirected to text file that doesn't add non-printable characters

**jotta-cli status (`jotta-cli status`):**
```

```

**Relevant logs for the issue (`cat ~/.jottad/jottabackup.log`)**
```

```

**Traceback**
```

```

**Additional info:**

**Comment by Kimbsen** [comment]
Will fix this for the next release!

**Comment by Swallowtail23** [comment]
Awesome, thank you

**Comment by Kimbsen** [comment]
We have released version 0.6.23861 which fixes this

**Comment by Swallowtail23** [comment]
I've updated one machine... it now appears to be stuck (+24 hours):

[root@emp86 ~]# jotta-cli status
------------------------------------------------------------------------------
 Account   : simon@simonandkate.net on Jottacloud
 Usage     : 2.31TiB / ( Unlimited )
 Device    : emp86
 Backups   :
------------------------------------------------------------------------------
   Path      : /mnt/data/backups
   Status    : Waiting for scan to start

------------------------------------------------------------------------------
   Path      : /mnt/data/cardimg
   Status    : Waiting for scan to start

------------------------------------------------------------------------------
   Path      : /mnt/data/hordevfs
   Files     : 1 files / 7bytes
   Status    : Up to date!

------------------------------------------------------------------------------
   Path      : /mnt/data/img
   Status    : Waiting for scan to start

------------------------------------------------------------------------------
OK

Log:

pid:7026 2020/02/12 20:13:38 Starting /usr/bin/jottad version  
0.6.23861 as 987(jottad)/983
pid:7026 2020/02/12 20:13:38 jottad.init
pid:7026 2020/02/12 20:13:38 jottad.init.appdata => /var/lib/jottad
pid:7026 2020/02/12 20:13:38 jottad.appdata     /var/lib/jottad
pid:7026 2020/02/12 20:13:38 jottad.appdata      
/var/lib/jottad/.ignore (1579176239|490)
pid:7026 2020/02/12 20:13:38 jottad.appdata     /var/lib/jottad/db  
(1581343134|8699904)
pid:7026 2020/02/12 20:13:38 jottad.appdata     /var/lib/jottad/downloads
pid:7026 2020/02/12 20:13:38 jottad.appdata      
/var/lib/jottad/downloads/history (1579176240|16384)
pid:7026 2020/02/12 20:13:38 jottad.appdata      
/var/lib/jottad/jottabackup.log (1581502418|1077821)
pid:7026 2020/02/12 20:13:38 jottad.appdata     /var/lib/jottad/nstore  
- (863311cbbfd4b2a2a92b2d583c3f7862|1581499118|65536)
pid:7026 2020/02/12 20:13:38 jottad.appdata      
/var/lib/jottad/queue.10.q (1581343134|16384)
pid:7026 2020/02/12 20:13:38 jottad.appdata      
/var/lib/jottad/userconfig.json (1581335914|185)
pid:7026 2020/02/12 20:13:38 jottad.init.cfg.downloadratae => 0B
pid:7026 2020/02/12 20:13:38 jottad.init.cfg.uploadrate => 0B
pid:7026 2020/02/12 20:13:38 jottad.init.cfg.scaninterval =>  1h0m0s
pid:7026 2020/02/12 20:13:38 jottad.init.user =>  
[pcrlxfcbrgxryzhkac015lbq] on [emp86] @  
[ad2d22029e87baa447fedbc71d26512a]
pid:7026 2020/02/12 20:13:38 jottad.init.loadedbackup  
/mnt/data/backups => backups
pid:7026 2020/02/12 20:13:38 jottad.init.loadedbackup  
/mnt/data/cardimg => cardimg
pid:7026 2020/02/12 20:13:38 jottad.init.loadedbackup  
/mnt/data/hordevfs => hordevfs
pid:7026 2020/02/12 20:13:38 jottad.init.loadedbackup /mnt/data/img => img
pid:7026 2020/02/12 20:13:38 jottad.init.done => ok
pid:7026 2020/02/12 20:13:38 jottad.startup
pid:7026 2020/02/12 20:13:38 jottad.startup.confirm-remote
pid:7026 2020/02/12 20:13:38 api.grpc.active => 127.0.0.1:14443
pid:7026 2020/02/12 20:13:38 running 3 delayed startup tasks
pid:7026 2020/02/12 20:13:38 updating account info true true
pid:7026 2020/02/12 20:13:41 jottad.startup.confirm-remote => ok
pid:7026 2020/02/12 20:13:41 jottad.startup => ok
pid:7026 2020/02/12 20:13:41 Queueing scan of added path /mnt/data/hordevfs
pid:7026 2020/02/12 20:13:41 Queueing scan of added path /mnt/data/cardimg
pid:7026 2020/02/12 20:13:41 Queueing scan of added path /mnt/data/img
pid:7026 2020/02/12 20:13:41 Queueing scan of added path /mnt/data/backups
pid:7026 2020/02/12 20:13:41 ok stat with 0 got filesystem info  
id[9210070729dffe91] type[ext2/ext3] for path /mnt/data/hordevfs
pid:7026 2020/02/12 20:13:41 Running scan of  
[9210070729dffe91/9210070729dffe91] /mnt/data/hordevfs
pid:7026 2020/02/12 20:13:41 Scan of /mnt/data/hordevfs completed in  
482.816µs. Found 1 files 7bytes
pid:7026 2020/02/12 20:13:42 GET https://repo.jotta.us/archives/VERSION 200

---- Message from Kimbsen <notifications@github.com> ---------
    Date: Mon, 10 Feb 2020 03:31:59 -0800

**Comment by Kimbsen** [comment]
@Swallowtail23 we are looking into this.

**Comment by Swallowtail23** [comment]
3 days later, the service is still waiting for scan to start, so I can only assume that the PC is not getting synced so my data is not getting backed up. Have you been able to replicate?

**Comment by Kimbsen** [comment]
We've been able to replicate it. Its a racecondition during completion of scans. It probably won't happen every time so a restart might allow backups to move forward. But we will release a fix as soon as we can. Probably today or tomorrow.

**Comment by Kimbsen** [comment]
We just released version 0.6.24236 which fixes this.

**Comment by Swallowtail23** [comment]
Can confirm fixed for me.
Thank you.

Simon Wilson
M: 0400 121 116

-------------------------
FROM: Kimbsen <notifications@github.com>
SENT: Tuesday, 18 February 2020 10:19 pm
TO: jotta-cli-issues@noreply.github.com
CC: Swallowtail23; Mention
SUBJECT: Re: [jotta/jotta-cli-issues] Recent change adding non-printable
characters to jotta-cli status redirected output (#103)

> We just released version 0.6.24236 which fixes this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub[1], or unsubscribe[2].
> [ { "@context": "http://schema.org", "@type": "EmailMessage",
> "potentialAction": { "@type": "ViewAction", "target":
>
"https://github.com/jotta/jotta-cli-issues/issues/103?email_source=notifications\u0026email_token=AB6VM4B7YSJOSNCENFFHHXTRDPG4RA5CNFSM4KCI7X72YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEMBYL7Y#issuecomment-587433471",
"url":
"https://github.com/jotta/jotta-cli-issues/issues/103?email_source=notifications\u0026email_token=AB6VM4B7YSJOSNCENFFHHXTRDPG4RA5CNFSM4KCI7X72YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEMBYL7Y#issuecomment-587433471",
"name": "View Issue" }, "description": "View this Issue on GitHub",
"publisher": { "@type": "Organization", "name": "GitHub", "url":
"https://github.com" } }
> ]



Links:
------
[1]  
https://github.com/jotta/jotta-cli-issues/issues/103?email_source=notifications&email_token=AB6VM4B7YSJOSNCENFFHHXTRDPG4RA5CNFSM4KCI7X72YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEMBYL7Y#issuecomment-587433471
[2]
